addons/rose/index.js

```diff
@@ -9,38 +9,14 @@ module.exports = {
   included(app) {
     this._super.included.apply(this, arguments);
 
-    this.includeStyles(app);
+    app.import('node_modules/codemirror/lib/codemirror.css');
+    app.import('node_modules/codemirror/theme/monokai.css');
```

Comment:
Nice, this is much cleaner!

Reply:
Does this remove the need to import these css files in our other `ember-cli-build` files as well?

Reply:
Yup.  Ideally Rose brings all the styles and assets along with it, with no additional configuration.  It's been a journey.

Reply:
Note that this PR handles the removal of these imports from admin and desktop.